### DISCUSSION TITLE
Change ICON size of the Profile Photo of IA

### DISCUSSION THREAD

**Discussion by Cometakah** [open-webui/open-webui#3807] [Q&A]
Hi, i don't speak English so good (I speak Spanish, sorry for the bad drafting)... But, i need help to make posible one idea...

I'm trying to change the size of the pictures of the Open-webui for aesthetic purposes. (The default icon size is too small)
I know it can be solved by changing the code a little bit, but I'm afraid of remove something and make an irreparable mistake.

![Foto 1](https://github.com/user-attachments/assets/1b3972b9-3b21-4b76-aa90-096c8877bce3)

![Foto 2](https://github.com/user-attachments/assets/3948d638-69f5-4eb1-aa43-724af1c29043)

If you can change the format from circular to square I would appreciate it, but it is not strictly necessary.

So if anyone knows what steps i nedd to follow to make the idea a reality, I would greatly appreciate it. Thnx

**Comment by WizenThorne** [comment]
It wouldn't be the worst thing in the world to add a few options.

**Comment by envelopingCODE** [comment]
+1 Yes, please
